spec.html

```diff
@@ -12299,6 +12299,26 @@ <h1>
       1. Return ~unused~.
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-canbeheldweakly" type="abstract operation">
+    <h1>
+      CanBeHeldWeakly (
+        _v_: an ECMAScript language value,
+      ): a Boolean
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It returns *true* if and only if _v_ is considered to have an identity suitable for a weak reference. i.e. Is _v_ a valid candidate key of a WeakMap, or entry in a WeakSet, or target of a WeakRef or FinalizationRegistry.</dd>
+    </dl>
+    <emu-alg>
+      1. If _v_ is an Object, return *true*.
+      1. If _v_ is a Symbol and KeyForSymbol(_v_) is *undefined*, return *true*.
+      1. Return *false*.
+    </emu-alg>
+    <emu-note type="editor">
```

Comment:
I'm guessing there shouldn't be editor's notes at Stage 4. Convert to a regular note, or drop it?